### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by akhmelov**
Created menu

### CODE REVIEW

File: src/templates/index.jade [hunk 1]
```diff
@@ -8,6 +8,37 @@ html
...
+                    li.name
+                        a(href='#venue') VENUE
+                    li.name
+                        a(href='#tutors') TUTORS
+                    li.name
+                        a(href='#sponsors') SPONSORS
+                    li.name
+                        a(href='#organizers') TEAM
+                    li.name
+                        a(href='https://twitter.com/codepot_pl') &nbsp;
```
Comment (mjasion) [inline src/templates/index.jade]: add `target='_blank'` for opening link in new tab

File: src/templates/index.jade [hunk 2]
```diff
@@ -8,6 +8,37 @@ html
...
+                            i.icon-twitter
+                    li.name
+                        a(href='https://www.facebook.com/pages/Codepot/397791273715096') &nbsp;
```
Comment (mjasion) [inline src/templates/index.jade]: add `target='_blank'` for opening link in new tab

File: src/templates/index.jade [hunk 3]
```diff
@@ -8,6 +8,37 @@ html
...
+                            i.icon-facebook-1
+                    li.name
+                        a(href='http://www.meetup.com/Warsaw-Codepoters/') &nbsp;
```
Comment (mjasion) [inline src/templates/index.jade]: add `target='_blank'` for opening link in new tab

File: src/sass/style.scss [hunk 4]
```diff
@@ -1,4 +1,5 @@
 @import "foundation/functions";
+@import "foundation";
```
Comment (neciu) [inline src/sass/style.scss]: `foundation.css` imports **all** foundation components. We don't want to do this. Import only top-bar component and move it below.

File: src/sass/style.scss [hunk 5]
```diff
@@ -143,6 +146,31 @@ section {
   }
 }
 
+nav {
```
Comment (neciu) [inline src/sass/style.scss:174]: Most of your changes can be achieved by providing variables for top bar component. Check `_top-bar.scss` file.

Note that variables you override should be **before** component import statement.

File: src/sass/style.scss [hunk 6]
```diff
@@ -186,7 +214,7 @@ section#about {
   }
 }
 
-section#map-information {
+section#venue {
```
Comment (neciu) [inline src/sass/style.scss:251]: cool

File: src/templates/index.jade [hunk 7]
```diff
@@ -8,6 +8,37 @@ html
         link(rel='stylesheet' type='text/css' href='public/style.css')
         script(src='public/head.js')
     body(itemtype='http://schema.org/EducationEvent', itemscope)
+        nav.top-bar(data-topbar role='navigation')
```
Comment (neciu) [inline src/templates/index.jade]: looks ok :+1:

File: src/templates/index.jade [hunk 8]
```diff
@@ -8,6 +8,37 @@ html
...
+            ul.title-area
+                li.name
+                    h1
```
Comment (neciu) [inline src/templates/index.jade]: Missing title?

**Comment by neciu** [comment]
One more note. Please make nav bar fixed.

File: src/templates/index.jade (hunk 8 above)
Comment (akhmelov) [inline src/templates/index.jade]: There is empty title. Because title area is important when page is opened on mobile.

**Pull request merged**